pkg/status/status.go

```diff
@@ -35,3 +35,8 @@ func GenerateBuildMsg(stage Stage, state State, msg string) string {
 	statusMsg := []string{string(stage), string(state), msg}
 	return strings.Join(statusMsg, "|")
 }
+
+func GenerateWorkflowStatusMsg(stepPosition int32, state State, errorCode string, msg string) string {
```

Comment:
also change to `errorCode, msg string` no need to define types in 2 consecutive arguments of the same type

Reply:
ok